doc/cprover-manual/contracts-decreases.md

```diff
@@ -1,3 +1,183 @@
-# Decreases Clause
+# Decreases Clauses
 
-TODO: Document `__CPROVER_decreases`
+A _decreases_ clause specifies a measure that must strictly decrease
+monotonically at every iteration of a loop.
```

Comment:
I would simply say "must strictly decrease at every iteration" rather than "strictly decrease monotonically at every iteration". Monotonically decreasing means decreases or remains the same, but never increasing. Strictly decreasing is stronger so although adding "monotonically" doesn't alter the meaning it's presence only serves to add a bit of doubt as to what you meant.

Reply:
Yes, "monotonically decreases" / "decreases monotonically" means what you mentioned, but "strictly decreases monotonically" means a measure decreases consistently (doesn't remain the same / increases) and [is widely used](https://www.google.com/search?q=%22strictly+decreases+monotonically%22). I don't think it's incorrect, but I agree that it may be confusing, so I'll change this as you suggested.